src/pages/library/RequestDatasetAccessModal.js

```diff
@@ -12,111 +12,104 @@ import * as Utils from 'src/libs/utils'
 
 
 const sendCopyEnabled = false
+const requestAccessEnabled = false
 
 export const RequestDatasetAccessModal = ({ onDismiss, datasets }) => {
   const [reason, setReason] = useState('')
   const [sendCopy, setSendCopy] = useState(false)
+  const [showWipModal, setShowWipModal] = useState(false)
 
-  return h(Modal, {
-    title: 'Request Access',
-    width: '40rem',
+  return showWipModal ? h(Modal, {
+    title: `Keep in mind we're still in BETA`,
```

Comment:
Any reason to use a template string here?

Reply:
no, i can change them back! :o